describe/requirements.txt

```diff
@@ -1,5 +1,27 @@
-mlrun
-sklearn
-matplotlib
-seaborn
-scikit-plot
\ No newline at end of file
+PyYAML~=5.4.1
+Jinja2~=3.0.3
+click~=8.0.4
+bs4~=0.0.1
+beautifulsoup4~=4.10.0
+Sphinx~=3.5.4
+black~=22.1.0
+mlrun~=0.10.0
+pandas~=1.3.5
+plotly~=5.6.0
+pytest~=7.0.1
+sklearn~=0.0
+scikit-learn~=1.0.2
+cloudpickle~=2.0.0
+numpy~=1.21.5
+matplotlib~=3.5.1
+SQLAlchemy~=1.4.31
+requests~=2.27.1
+Pillow~=9.0.1
+seaborn~=0.11.2
+scipy~=1.7.3
+pyarrow~=5.0.0
+six~=1.16.0
+v3io~=0.5.15
+storey~=0.10.5
```

Comment:
I think you over done it. Most of the packages here are just requirements of `mlrun`, so putting `mlrun` is enough. Shrink it for all that is necessary that you have `pip install`ed.